netcontrold/tests/lib/test_prelim.py

```diff
@@ -0,0 +1,68 @@
+#
+#  Copyright (c) 2020 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+from unittest import TestCase
+
+from netcontrold.lib import dataif
+from netcontrold.lib.error import NcdException,ObjConsistencyExc
+
+class TestDataif_add_rxq(TestCase):
+
+	def setUp(self):
+		self.port1= dataif.Port("port1")
+
+
+	def test_port_add_rxq_1(self):
```

Comment:
Comment with one or two lines of what a test case is about is helpful for anyone doing code walk. Applicable for all the tests written.